packages/rest/src/lib/CDN.ts

```diff
@@ -158,7 +187,11 @@ export class CDN {
 	 * @param extension The extension of the sticker
 	 */
 	public sticker(stickerId: string, extension?: StickerExtension): string {
-		return this.makeURL(`/stickers/${stickerId}`, { allowedExtensions: ALLOWED_STICKER_EXTENSIONS, extension });
+		// Stickers cannot have a `.webp` extension
+		return this.makeURL(`/stickers/${stickerId}`, {
+			allowedExtensions: ALLOWED_STICKER_EXTENSIONS,
+			extension: extension ?? 'png', // Stickers cannot have a `.webp` extension, so we default to a `.png`
+		});
```

Comment:
No need to mention it twice
```suggestion
		return this.makeURL(`/stickers/${stickerId}`, {
			allowedExtensions: ALLOWED_STICKER_EXTENSIONS,
			extension: extension ?? 'png', // Stickers cannot have a `.webp` extension, so we default to a `.png`
		});
```